rpm/build_rpms.bsh

```diff
@@ -14,7 +14,7 @@ else #Basically Centos 5/6
 fi
 
 case "${OS_NAME}" in
-  centos*|red*)
+  centos*|red*|almalinux)
```

Comment:
I don't think this is required.  We don't actually call this for any builds except for `centos`.  I don't think it hurts anything to leave it, though.